x-pack/plugins/security_solution/public/detections/components/host_isolation/use_sentinelone_host_isolation.tsx

```diff
@@ -20,27 +21,63 @@ interface ErrorType {
   meta: ActionTypeExecutorResult<SentinelOneGetAgentsResponse>;
 }
 
+// TODO:
+// 8.14
+// Remove this function when
+// agentStatusClientEnabled is enabled/removed
 export const useGetSentinelOneAgentStatus = (
   agentIds: string[],
-  options: UseQueryOptions<AgentStatusApiResponse['data'], IHttpFetchError<ErrorType>> = {}
-): UseQueryResult<AgentStatusApiResponse['data'], IHttpFetchError<ErrorType>> => {
+  agentType?: string,
+  options: UseQueryOptions<AgentStatusInfo, IHttpFetchError<ErrorType>> = {}
+): UseQueryResult<AgentStatusInfo, IHttpFetchError<ErrorType>> => {
   const http = useHttp();
 
-  return useQuery<AgentStatusApiResponse['data'], IHttpFetchError<ErrorType>>({
+  return useQuery<AgentStatusInfo, IHttpFetchError<ErrorType>>({
     queryKey: ['get-agent-status', agentIds],
     ...options,
     // TODO: update this to use a function instead of a number
     refetchInterval: 2000,
     queryFn: () =>
       http
-        .get<{ data: AgentStatusApiResponse['data'] }>(ENDPOINT_AGENT_STATUS_ROUTE, {
+        .get<{ data: AgentStatusInfo }>(AGENT_STATUS_ROUTE, {
           version: '1',
           query: {
             agentIds,
             // 8.13 sentinel_one support via internal API
-            agentType: 'sentinel_one',
+            agentType: agentType ? agentType : 'sentinel_one',
           },
         })
         .then((response) => response.data),
   });
 };
+
+export const useGetAgentStatus = (
+  agentIds: string[],
+  agentType: string,
+  options: UseQueryOptions<AgentStatusRecords, IHttpFetchError<ErrorType>> = {}
+): UseQueryResult<AgentStatusRecords, IHttpFetchError<ErrorType>> => {
+  const http = useHttp();
+
+  return useQuery<AgentStatusRecords, IHttpFetchError<ErrorType>>({
+    queryKey: ['get-agent-status', agentIds],
+    ...options,
+    // TODO: remove this refetchInterval and instead override it where called, via options.
+    refetchInterval: 2000,
```

Comment:
wondering if we should increase this to 5s. Current value seems to short to me. Thoughts?

Also - can you move this above line 63? Doing so will allow calls of this hoot to override this interval.

Reply:
842aec33ebf16bb628c93be09bc47b8ead49be0b